process/transaction/shardProcess.go

```diff
@@ -393,6 +409,229 @@ func (txProc *txProcessor) moveBalances(
 	return nil
 }
 
+func (txProc *txProcessor) processRelayedTx(
```

Comment:
This needs heavy testing

Reply:
agreed after first reviews will do it. may merge after disabling.